src/main/scala/cognite/spark/v1/AssetsRelation.scala

```diff
@@ -160,7 +160,14 @@ final case class AssetsReadSchema(
     createdTime: Instant = Instant.ofEpochMilli(0),
     lastUpdatedTime: Instant = Instant.ofEpochMilli(0),
     rootId: Option[Long] = Some(0),
-    aggregates: Option[Map[String, Long]] = None,
+    aggregates: Option[AssetsAggregatesSchema] = None,
```

Comment:
Why does this need to change? Or maybe it doesn't, but you're doing more than just adding support for Scala 2.13?

In that case, could we have a smaller PR with only the changes necessary for Scala 2.13, please?

Reply:
The changes to SparkSchemaHelper are quite large and invasive, maybe they can be smaller if `immutable.Seq` instead of `Iterable` helps. I haven't really tried to understand all those changes, or why they would be necessary just for 2.13.

Reply:
I don't think that will help and Iterable makes it less picky which kind of `Seq` are we using (I think that it would mostly remove just the `.toVector` call)